src/components/ChallengeEditor/ChallengePrizes-Field/index.js

```diff
@@ -84,7 +84,7 @@ class ChallengePrizesField extends Component {
             }
           </div>)}
         </div>
-        {!readOnly && (prize.value === '' || (+prize.value <= 0 || +prize.value > 1000000)) && (
+        {!readOnly && challenge.submitTriggered && (prize.value === '' || (+prize.value <= 0 || +prize.value > 1000000)) && (
```

Comment:
@maxceem can we get this fix via form's state i.e. when the state is pristine, don't show the validation error. I am asking this because it is the standard way we are using it in Connect. However, I am okay with the way you fixed it right now in case none of the other fields is using that form state to hide the error without user making the field dirty which I think is the case as I can't see any field showing the validation error without hitting submit button.

Reply:
@vikasrohit Right, all of the other fields use this approach - validate on submit only. So I use the same approach here for consistency.